OIIInspector/ImageManager.py

```diff
@@ -0,0 +1,151 @@
+import logging
```

Comment:
Could you rename this file (and other files) to use lowercase+underscores instead of camelcase?